crates/nu-std/lib/help.nu

```diff
@@ -653,33 +653,44 @@ def show-command [command: record] {
     print ""
 }
 
-# Show help on nushell commands.
+# Show help on commands.
 export def "help commands" [
     ...command: string@"nu-complete list-commands"  # the name of command to get help on
     --find (-f): string  # string to find in command names and usage
 ] {
-    let commands = ($nu.scope.commands | where not is_extern | reject is_extern | sort-by name)
+    let nu_commands = ($nu.scope.commands | sort-by name | where not is_extern | reject is_extern )
 
     let command = ($command | str join " ")
 
     if not ($find | is-empty) {
-        let found_commands = ($commands | find $find --columns [name usage search_terms])
+        # TODO: impl find for external commands
+        let nu_commands = ($nu_commands | find $find --columns [name usage search_terms])
```

Comment:
i'm not sure i understand the renaming of `found_commands` to `nu_commands` and `commands` to `nu_commands` :thinking: 
just looks it makes the changes here more complicated than they need to be, right? :yum:

Reply:
Yea, it happened during dev, I can change these things back

Reply:
it would make the changes easier to deal with, i think it's a bit better :blush: